roles/mail.rb

```diff
@@ -46,6 +46,11 @@
         :comment => "join.osmfoundation.org",
         :domains => ["join.osmfoundation.org"],
         :host => "ridley.ucl.openstreetmap.org"
+      },
+      :discourse => {
+        :comment => "community.osmfoundation.org",
+        :domains => ["community.osmfoundation.org"],
+        :host => "lockheed.ams.openstreetmap.org"
```

Comment:
That will get the email to lockheed, but how does it then get into discourse? The exim on lockheed will somehow need to route it to the discourse mail receiver container?

Reply:
The `mail-receiver` container runs postfix with discourse handler. [Standard config it runs on port 25](https://github.com/openstreetmap/chef/pull/483/files#diff-4b7f61680f19b3fe1dded83774a2388ebc80e0bdb661f92a1822a3f4490ce1f0R14), but these can be mapped to another port if exim would be able to handle that.

Reply:
Well it won't be able to bind port 25 because exim will already be there, unless you do something to either stop exim running or stop it listening on port 25.

Reply:
Added ::2500 and moved the docker external port to 2500 for the `mail-receiver`